modules/theme/src/themes/default/views/card.overrides

```diff
@@ -36,7 +39,20 @@
             }
         }
         .top-action-panel {
-            padding: 0 @settingsCardPadding;;
+            box-shadow: none;
+            padding: 0 @settingsCardPadding;
+            border-left: 0px;
+            border-right: 0px;
+            border-top: 1px solid #eee;
```

Comment:
Shall we get this value from a variable?

Reply:
This value was moved to the variable `settingsCardTopActionPanelBorderTop`